main.tf

```diff
@@ -0,0 +1,104 @@
+# Azure Provider source and version being used
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">=2.49"
+    }
+  }
+
+  # Stores the state as a Blob with the given Key within the Blob Container within the Blob Storage Account
+  backend "azurerm" {
+      resource_group_name   = "CreditSuisse1_PatrickMorris_ProjectExercise"
+      storage_account_name  = "tstatesan"
+      container_name        = "tstatecn"
+      key                   = "terraform.tfstate"
+  }
+}
+
+# Configure the Microsoft Azure Provider
+provider "azurerm" {
+  features {}
+}
+
+# Create a resource group
+data "azurerm_resource_group" "main" {
+  name = "CreditSuisse1_PatrickMorris_ProjectExercise"
+}
+
+resource "azurerm_app_service_plan" "main" {
+  name = "terraformed-asp"
+  location = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  kind = "Linux"
+  reserved = true
+  
+  sku {
+    tier = "Basic"
+    size = "B1"
+  }
+}
+
+resource "azurerm_app_service" "main" {
+  name = "${var.prefix}-todo-app-pm"
+  location = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  app_service_plan_id = azurerm_app_service_plan.main.id
+  
+  site_config {
+  app_command_line = ""
+  linux_fx_version = "DOCKER|pm141088/todo-app:latest"
+  }
```

Comment:
Run `terraform fmt` to fix the formatting

Reply:
Thank you :)